src/app/contextBroker/contextBroker.cpp

```diff
@@ -1554,6 +1561,14 @@ int main(int argC, char* argV[])
 
   while (1)
   {
-    sleep(10);
+    if (subCacheInterval != 0)
+    {
+      sleep(subCacheInterval);
+      // orion::subCache->refresh();
```

Comment:
This comment should include a FIXME and reference to the number of the issue about refresh()...